doc/auth.md

```diff
@@ -0,0 +1,78 @@
+# Enable Authentication
+
+Pravega supports pluggable authentication and authorization (referred to as auth for short). For details please see [Pravega Authentication](https://github.com/pravega/pravega/blob/master/documentation/src/docs/auth/auth-plugin.md).
+
+By default, the PasswordAuthHandler plugin is installed on the system.
+To use the default `PasswordAuthHandler` plugin for `auth`, the following steps can be followed:
+
+1. Create a file containing `<user>:<password>:<acl>;` with one line per user.
+Delimiter should be `:` with `;` at the end of each line.
+Use the   [PasswordCreatorTool](https://github.com/pravega/pravega/blob/master/controller/src/test/java/io/pravega/controller/auth/PasswordFileCreatorTool.java) to create a new file with the password encrypted.
```

Comment:
"password encrypted" is incorrect here. The password is actually hashed.

Reply:
The PasswordFileCreationTool mentions this as "encryption" and  I could not find any mention of this in pravega security documentation as to whether this is 'hashing' or 'encoding'. 
Could you please point me to the same?